### ISSUE TITLE
Batch files for Novelty Search

### ISSUE THREAD

**Issue by schrum2** [schrum2/EvoCraft-SCOPE#71]
We need a variety of batch files. Maybe not for every possibility, but for many. At least one for each diversity characterization and distance metric ... whatever makes the most sense to combine. Some should be for snakes and others for block volumes.

I agree it might make sense to experiment with decreased sizes as well ... make the XRANGE, etc in these tests smaller and see what happens. Keep an eye on what is generated ... it would be a shame if we generate a flying machine and somehow miss it!

**Comment by MuellMark** [comment]
Another thing to do is to reformat all of the batch files so that they're sorted like it is in MM-NEAT

**Comment by MuellMark** [comment]
Batch files for every combination of characteristic and distance metric has been made, as well as a way to load all of them. On top of this, There are batch files for snakes (and loading them), and also two batch files tailored to generating flying machines with a smaller generated box shape (and ways to load them also). I also reorganized the folder of batch files so everything is in a specified folder, just like it is in MM_NEAT. for this to be done, an extra cd.. was added to all batch files
